### PR TITLE
CE-3240: Script to revert flags

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Remove flags and add appropriate templates into article

### CODE REVIEW

File: extensions/wikia/Flags/maintenance/RevertNotice.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+$dir = dirname( __FILE__ ) . "/../../../../";
+require_once( $dir . 'maintenance/Maintenance.php' );
+
+use Flags\Models\Flag;
+use Flags\Models\FlaggedPages;
+
+class RevertNotice extends Maintenance {
+
+	const EDIT_SUMMARY = 'Moving back templates from [[Special:Flags]] feature.';
```
Comment (adamkarminski) [inline extensions/wikia/Flags/maintenance/RevertNotice.php]: I'd phrase it as: "Restoring templates used in the [[Special:Flags]] feature" but @Grunny - please confirm. :)

File: extensions/wikia/Flags/maintenance/RevertNotices.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	}
+
+	public function execute() {
+		global $IP;
+
+		$flagModel = new Flag();
+		$wikiIds = $flagModel->getWikisWithFlags();
+
+		foreach ( $wikiIds as $wikiId ) {
+			$cmd = "SERVER_ID=$wikiId /usr/bin/php {$IP}/extensions/wikia/Flags/maintenance/RevertNotice.php";
```
Comment (adamkarminski) [inline extensions/wikia/Flags/maintenance/RevertNotices.php]: Escape arguments with `wfEscapeShellArg()`

File: extensions/wikia/Flags/maintenance/RevertNotice.php (hunk 1 above)
Comment (Grunny) [inline extensions/wikia/Flags/maintenance/RevertNotice.php]: Yep, Adam's suggestion sounds good to me. :)

**Comment by adamkarminski** [comment]
LGTM

File: extensions/wikia/Flags/maintenance/RevertNotice.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+use Flags\Models\Flag;
+use Flags\Models\FlaggedPages;
+use Wikia\Util\GlobalStateWrapper;
+
+class RevertNotice extends Maintenance {
+
+	const EDIT_SUMMARY = 'Restoring templates used in the [[Special:Flags]] feature.';
+
+	public function __construct() {
```
Comment (Grunny) [inline extensions/wikia/Flags/maintenance/RevertNotice.php]: This constructor isn't changing anything, so is unnecessary.

File: extensions/wikia/Flags/maintenance/RevertNotices.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+$dir = dirname( __FILE__ ) . "/../../../../";
+require_once( $dir . 'maintenance/Maintenance.php' );
+
+use Flags\Models\Flag;
+
+class RevertNotices extends Maintenance {
+	public function __construct() {
```
Comment (Grunny) [inline extensions/wikia/Flags/maintenance/RevertNotices.php]: Same here.

**Comment by Grunny** [comment]
LGTM.

**Comment by azarek** [comment]
LGTM!! :) Good job! You did great! Awesome script! Work beautiful! Perfect!

**Comment by lukaszkonieczny** [comment]
:blush:

**Pull request merged**